services/QuillLMS/engines/evidence/lib/evidence/engine.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 # require_relative '../quill_scaffold_controller'
+require 'factory_bot_rails'
```

Comment:
Since this is development_dependency we probably want guard `unless Rails.env.production?`